### ISSUE TITLE
How to get all strings that contains any of a given words?

### ISSUE THREAD

**Issue by ComptonAlvaro** [dotnet/efcore#19273]
I am using EF Core 3.0.

I am trying to get all the rows that a property contains any of a list of strings. I am trying in this way:

```C#
string myWordsToSearch = "word1 word2 word3";
List<string> miLstWords = myWordsToSearch
    .Split(" ")
    .Where(x => string.IsNullOrWhiteSpace(x) == false)
    .ToList();
MyDbContext.MyDbSet.Where(x => miLstWords.Any(y => x.MyPropertyInDbSet.Contains(y)));
```

But I get this exception:

`    One or more errors occurred. (The LINQ expression 'Where( source: DbSet, predicate: (s) => Any( source: (Unhandled parameter: __miLstWords_0), predicate: (y) => MyPropertyInDbSet.Contains(y)))' could not be translated. Either rewrite the query in a form that can be translated, or switch to client evaluation explicitly by inserting a call to either AsEnumerable(), AsAsyncEnumerable(), ToList(), or ToListAsync(). See https://go.microsoft.com/fwlink/?linkid=2101038 for more information.`

Thanks.

**Comment by ajcvickers** [comment]
@ComptonAlvaro Please try EF Core 3.1. (This looks similar to issues that were fixed for 3.1, such as #17342)

**Comment by smitpatel** [comment]
Duplicate of #19070

**Comment by ComptonAlvaro** [comment]
I have tried with EF Core 3.1.0 and I still have  the same problem.

**Comment by ajcvickers** [comment]
@ComptonAlvaro #19070 is not fixed in 3.1--it's in the Backlog milestone. Are you saying that this is not a duplicate of #19070? If so, please post a small, runnable project that demonstrates what you are seeing so that we can understand the difference.
